docs/reference/api_examples.yml

```diff
@@ -249,6 +249,9 @@
 
   doc: |
     Get a list of all datasets in a deployment.
+    This includes manifest (attached) and non-manifest (unattached) datasets.
+    A dataset is non-manifest when it is moving between nodes.
```

Comment:
That's just one example of when it's non-manifest but this reads like it's comprehensive.